R/mobr_boxplots.R

```diff
@@ -143,7 +150,15 @@ These are removed for the calculation of rarefied richness."))
                         betaPIE = numeric(nperm)
                         )
    
-   for (i in 1:nperm){
+   F_rand$S[1] = F_obs$S
+   F_rand$N[1] = F_obs$N
+   F_rand$S_rare[1] = F_obs$S_rare
+   F_rand$PIE[1] = F_obs$PIE
+   F_rand$ENS_PIE[1] = F_obs$ENS_PIE
+   F_rand$S_asymp[1] = F_obs$S_asymp
+   F_rand$betaPIE[1] = F_obs$betaPIE
+   
+   for (i in 2:nperm){
```

Comment:
here's another thread where this once came up if you are curious : https://github.com/vegandevs/vegan/pull/162#issuecomment-210513065